python/tvm/autotvm/measure/measure_methods.py

```diff
@@ -511,7 +523,13 @@ def __call__(self, measure_input, tmp_dir, **kwargs):
             )
             # TODO(tvm-team) consider linline _build_func_common
             func, arg_info = _build_func_common(measure_input, **kwargs)
-            func.export_library(filename, self.build_func)
+            if self.build_func.output_format == "tar.gz":
+                # Late import to preserve autoTVM with USE_MICRO OFF
+                from tvm import micro  # pylint: disable=import-outside-toplevel
```

Comment:
I think it makes sense here to `try:` import, rather than leave it to crash when `USE_MICRO OFF`. Also maybe add an error message.

Reply:
Agree. Fixed it.